test/test_matrix_poisson_solver.jl

```diff
@@ -0,0 +1,145 @@
+using Oceananigans.Solvers: solve!, MatrixIterativeSolver
+using Oceananigans.Fields: interior_copy
+using Oceananigans.Operators: volume, Δyᶠᶜᵃ, Δyᶜᶠᵃ, Δxᶠᶜᵃ, Δxᶜᶠᵃ, Δyᵃᶜᵃ, Δxᶜᵃᵃ, Δzᵃᵃᶠ, Δzᵃᵃᶜ, ∇²ᶜᶜᶜ
+using Oceananigans.Architectures: arch_array
+using KernelAbstractions: @kernel, @index
+using Statistics, LinearAlgebra
+
+
+@kernel function _∇²!(∇²f, grid, f)
+    i, j, k = @index(Global, NTuple)
+    @inbounds ∇²f[i, j, k] = ∇²ᶜᶜᶜ(i, j, k, grid, f)
+end
```

Comment:
Isn't this available in `utils_for_runtests.jl`?

Reply:
right! removed!